golem-test-framework/src/components/component_service/docker.rs

```diff
@@ -43,7 +42,6 @@ pub struct DockerComponentService {
 }
 
 impl DockerComponentService {
-    const NAME: &'static str = "golem_component_service";
```

Comment:
As much as possible I changed these to work with dynamic names, so there is less risk of name conflicts